### ISSUE TITLE
Combine power consumption

### ISSUE THREAD

**Issue by yumi-modding** [Mogli12/GearboxAddon#81]
Hello,
I'm trying to enhance combine behaviour in FS17 since in default FS17 engine only cutter requires some power but result is useless since you can still harvest at max speed when removing speedLimit.
I would like to retrieve FS2013 moreRealistic behavior since I was a big fan of this engine.
I've seen in the GBAddon code that different power consumption can be set (threshing, chopper...) And so I've tested reusing MR values on the case IH 7130 combine.
I don't see a real difference and so I'd like to know if you think this part of the code could be enhanced.
I have the feeling that the different power values are well computed but then I lose the track when torques get into the game.
I'm comfortable with lua coding and so I can prototype some stuff if you already know some things to be fixed.
Thanks for your feedback.

**Comment by Mogli12** [comment]
Hi,

The power consumption stuff is already part of the gearbox addon since FS15. But for most machines I try to calculate some defaults. You find the coding in https://github.com/Mogli12/GearboxAddon/blob/master/src/gearboxMogli.lua lines 665ff.

But after a while I am not sure that this is the right way. I do not think that a combine harvester is operated at 100% load. In reality the loss would be much too high. So I guess that the loss of fruits limits the speed in reality and not missing motor power.

Regards, Stefan

**Comment by yumi-modding** [comment]
Hello,
Thanks for the comment I was looking at the right part of the code. I've set the values in config.xml not to use the default ones but I still feel to be limited only by cutter speed limit. Actually on MR2013 grain loss was implemented when too much power was required for threshing compared to the harvest speed. I'll try to see if I can tune some stuff.
For now I can propose some fix regarding changes occurred on FS17 (overloading spec to be used instead of pipeIsUnloading for ex)
